x-pack/plugins/session_view/public/components/process_tree/styles.ts

```diff
@@ -13,7 +13,7 @@ export const useStyles = () => {
   const { euiTheme } = useEuiTheme();
 
   const cached = useMemo(() => {
-    const defaultSelectionColor = euiTheme.colors.accent;
+    const defaultSelectionColor = 'rgb(0, 119, 204)';
```

Comment:
Does the design call for this? If design isn't using one of the eui theme colors it probably should. I usually refer to https://elastic.github.io/eui/#/theming/colors and search the hex code to find out which prop to use.

Reply:
Also the `defaultSelected` variable below can be removed since they are the same value, and then can update references that uses `defaultSelected` to use `defaultSelectionColor`